apps/client/src/app/core/data-reporting/exploration-result.reporter.ts

```diff
@@ -0,0 +1,8 @@
+export enum ExplorationType {
+  AIRSHIP,
+  SUBMARINE
+}
```

Comment:
Even if it may seem a bit too much, one file per export is a general rule i'm trying to follow (well, sometimes I don't, but it's rare), I'd love this to be in its own file.

Reply:
For now, I have put it inside `model/other`. Might actually be better to move it in a `airship/submarine` module tho.